pmlb/update_dataset_readmes.py

```diff
@@ -171,7 +87,8 @@ def write_readme(dataset, problem_type, df):
     }
 
     # figure out which datasets have changed for this commit
-    updated_datasets = get_updated_datasets()
+    # updated_datasets = get_updated_datasets()
+    updated_datasets = regression_dataset_names + classification_dataset_names
```

Comment:
Is this supposed to be a temporary change? I think we eventually would want these readmes to be automatically generated with CI as well. But at the same time, it doesn't take that long to generate each READMEs so I don't mind it rerun for all datasets when there's a merge to master.

Reply:
I don't think we need the readmes to be a part of CI anymore since they are just linking to static files. The only time they would be needed is if the dataset name changes and its pandas profiling location moves. AFAIK the readmes are no longer a part of the CI process anyway.

Reply:
OK that's fine for now. I just wanted to make sure if new datasets are added, they have readmes. But we can do this later.

Reply:
yea! that should be a requirement for a PR. maybe we should add that to the Contributing guide